### PR TITLE
Updates Bevy-0.13

### PR DESCRIPTION
**Pull request by naomijub**
- Adds CI
  - formatting and linter
  - tests??
- Formats code
- Minor enhancements

### CODE REVIEW

**Comment by romenjelly** [comment]
Thank you for this pull request!
Sorry for the late update, I was away from Bevy-related projects for a while.

My only real concern merging this is that the code changes and formatting changes are in the same commit, making it difficult to distinguish between what is change, and what is formatting. It is my fault for not providing a contributing guide, but nevertheless, thank you for your work!

Never having used `cargo fmt`, I suspect I should for public-facing projects.
Formatting such as systems almost always having its parameters on new lines is done purely to minimize potential git diff, since they are the most susceptible to change rapidly. However, since `cargo fmt` is now in the CI pipeline, it should no longer be an issue.

Before I merge, there are a few small changes required for this PR, which I will add specific comments for shortly.

File: examples/render_layers.rs [hunk 1]
```diff
@@ -123,26 +111,26 @@ fn setup(
...
-                clear_color: ClearColorConfig::Custom(Color::GRAY),
-                ..default()
-            },
-            camera: Camera {
-                order: -1,
-                target: RenderTarget::Image(bottom_image_handle.clone()),
+    let secondary_camera_entity = commands
+        .spawn((
+            Camera3dBundle {
+                camera_3d: Camera3d::default(),
```
[the remark this answers is not in the document]
Comment (naomijub) [reply examples/render_layers.rs]: bf39758

File: examples/render_layers.rs [hunk 2]
```diff
@@ -93,11 +83,9 @@ fn setup(
     // Top render layer camera
     commands.spawn((
         Camera3dBundle {
-            camera_3d: Camera3d {
-                clear_color: ClearColorConfig::Custom(Color::GRAY),
-                ..default()
-            },
+            camera_3d: Camera3d::default(),
```
[the remark this answers is not in the document]
Comment (naomijub) [reply examples/render_layers.rs]: bf39758

**Comment by naomijub** [comment]
Sorry, my code migration tool does formatting and linting with the migration, so it got all in the same commit

**Comment by naomijub** [comment]
Can you please publish version 0.5 @romenjelly ?

**Comment by romenjelly** [comment]
Oh, completely forgot!
Will do ASAP.

**Comment by naomijub** [comment]
Thanks

**Pull request merged**